src/main/java/core/basesyntax/ChangeToCar.java

```diff
@@ -1,9 +1,17 @@
 package core.basesyntax;
 
+import java.util.function.Function;
+
 /**
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод, который принимает вес авто
  * и возвращает тип авто в зависимости от веса (если weight меньше 1500, то CAR, если больше, то
  * TRUCK). Метод должен принимать Integer и возвращать AutoType</p>
  */
-public class ChangeToCar {
+public class ChangeToCar implements Function<Integer, AutoType> {
+    static final int MIN_TRUCK_WEIGHT = 1500;
```

Comment:
what modifiers should we use with constants?

Reply:
public. Done